### ISSUE TITLE
SVG images not rendering if src URL has no .svg extension

### ISSUE THREAD

**Issue by mplgn** [daohoangson/flutter_widget_from_html#1302]
Thanks for this awesome package!   

Images stored in my CMS (ButterCMS) have urls _without_ the image type file extension (e.g., no ".png", ".svg") in the `src` URL.     

- This works fine for all image types except svg.    
- When an svg image is give as the src, the alt text is rendered instead of the svg image
- An image url with the .svg extension does render correctly (unfortunately not an option for me because of the CMS)   

## Steps to Reproduce

<details>
  <summary>HTML</summary>

```html
<img src="https://cdn.buttercms.com/gGgFu5YTmWA3vwTcJGSM" alt="SVG Image">
```
</details>

<details>
  <summary>`HtmlWidget` configuration</summary>

```dart
HtmlWidget(
      '<img src="https://cdn.buttercms.com/gGgFu5YTmWA3vwTcJGSM" alt="SVG Image">',
)
```
</details>

<details>
  <summary>Tesing environment</summary>

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.22.3, on macOS 14.5 23F79 darwin-arm64, locale en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 15.4)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2024.1)
[✓] IntelliJ IDEA Ultimate Edition (version 2024.1.4)
[✓] VS Code (version 1.91.1)
[✓] Connected device (6 available)
[✓] Network resources

• No issues found!
```

```
```
</details>

## Expected results

<img width="529" alt="image" src="https://github.com/user-attachments/assets/f6811c9e-a51e-428a-af50-bc5dadb21611">

## Actual results

<img width="529" alt="image" src="https://github.com/user-attachments/assets/1c64595b-95db-4f16-be4a-e30d36e31cc0">

**Comment by daohoangson** [comment]
Yes, this is a limitation of the IMG handling logic. It depends on the extension to switch between built-in Flutter rendering and SVG rendering.
